### PR TITLE
User can see actual form fields

### PR DESCRIPTION
**Pull request by yuki24**
finishes https://artsyproduct.atlassian.net/browse/PURCHASE-337

This PR depends on the following PRs:

 * https://github.com/artsy/reaction/pull/1070
 * https://github.com/artsy/reaction/pull/1072
 * https://github.com/artsy/reaction/pull/1073

I haven't consolidated components yet but am open to feedback and suggestions!

# Screenshot

## Desktop

screen shot 2018-08-01 at 6 47 28 am

## Mobile

screencapture-localhost-9001-2018-08-01-06_52_29

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 92594d1db2a652575a25a177140c54d21fe83b9e

https://deploy-preview-1074--artsy-reaction.netlify.com

**Comment by zephraph** [comment]
It's so beautiful

**Comment by yuki24** [comment]
I'll take the 😄 emoji from @sweir27 as "merge on green"!

File: src/Apps/Order/Routes/Shipping/index.tsx [hunk 1]
```diff
@@ -38,34 +43,64 @@ export class ShippingRoute extends Component<ShippingProps> {
...
+                        placeholder="Add State, province, or region"
+                        title="State, province, or region"
+                        block
+                      />
+                    </Flex>
                   </TwoColumnSplit>
-                  {!xs && (
-                    <Link to={`/order2/${order.id}/payment`}>
-                      <Button size="large" width="100%">
-                        Continue
```
Comment (sweir27) [inline src/Apps/Order/Routes/Shipping/index.tsx]: Can we please add this button back? I know right now it's just a dummy that links to the next page, but it's really helpful for demonstrating routing behavior.

Comment (yuki24) [reply src/Apps/Order/Routes/Shipping/index.tsx]: I was actually wondering why this was gone. This was removed by accident and I'll add it back.

**Pull request merged**